.github/workflows/ci.yaml

```diff
@@ -87,7 +87,7 @@ jobs:
         uses: charmed-kubernetes/actions-operator@main
         with:
           provider: lxd
-          bootstrap-options: "--agent-version 2.9.29"
+          bootstrap-options: "--agent-version 2.9.38"
```

Comment:
thoughts on dropping the `bootstrap-options` config? i believe [this](https://github.com/charmed-kubernetes/actions-operator/blob/main/src/bootstrap/index.ts#L144) would bootstrap the latest juju version (3.1.0?)

Reply:
I see it otherwise, we should always use the latest Juju patchset of 2.9 release.
Also, we should detect ASAP the issues with newly released Juju
(while juju team is in context of solving regressions of newly released component).
I do not see the silent upgrades possibilities, as juju release is loudly announced.

Yes, there is a chance to restore pinning right after 2.9.39 release.
Let's see it on the practice.

Reply:
After checking the status in all our repos and the previous experience... I have changed my mind.
Let's keep juju-agent pinned until we trust in our tests stability to avoid PRs blocking in a day of Juju release.